sdk/servicebus/azure-servicebus/tests/async_tests/test_sb_client_async.py

```diff
@@ -661,11 +662,11 @@ async def test_custom_endpoint_connection_verify_exception_async(self,
         hostname = f"{servicebus_namespace.name}{SERVICEBUS_ENDPOINT_SUFFIX}"
         credential = AzureNamedKeyCredential(servicebus_namespace_key_name, servicebus_namespace_primary_key)
 
-        client = ServiceBusClient(hostname, credential, connection_verify="cacert.pem", uamqp_transport=uamqp_transport)
+        certfile = certifi.where()
+        client = ServiceBusClient(hostname, credential, connection_verify=certfile, uamqp_transport=uamqp_transport)
         async with client:
-            with pytest.raises(ServiceBusError):
```

Comment:
if we are going to make this so it doesn't raise an error can we change the test name b/c this is no longer for an exception